README.rst

```diff
@@ -240,7 +240,7 @@ License
    :target: https://scrutinizer-ci.com/g/coala-analyzer/coala/?branch=master
```

Comment:
the commit message implies that you are changing the readthedocs domain name, not fixing the outdated links in different files, please change the shortlog accordingly so that it explains exactly the change done :)